includes/emails/actions.php

```diff
@@ -16,19 +16,23 @@
  * Triggers Purchase Receipt to be sent after the payment status is updated
  *
  * @since 1.0.8.4
- * @param int $payment_id Payment ID
+ * @since 2.8 - Add parameters for EDD_Payment and EDD_Customer object.
+ *
+ * @param int          $payment_id Payment ID.
+ * @param EDD_Payment  $payment    Payment object for payment ID.
+ * @param EDD_Customer $customer   Customer object for associated payment.
  * @return void
  */
-function edd_trigger_purchase_receipt( $payment_id ) {
+function edd_trigger_purchase_receipt( $payment_id, $payment, $customer ) {
```

Comment:
We need to set some defaults here, since we're adding parameters that haven't existed before.

Just in case someone is calling this directly, even if we just set them to `null` so they trigger the `gets` in the end result of the action.